x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -70,3 +76,60 @@ export const StyledTime = memo(styled('time')`
   display: inline-block;
   text-align: start;
 `);
+
+interface StyledCopyableField {
+  readonly borderColor: string;
+  readonly hoverBackground: string;
+}
+
+const StyledCopyableField = styled.div<StyledCopyableField>`
+  border: 0.1em solid;
+  border-color: ${(props) => props.borderColor};
+  border-radius: 3px;
+  padding: 4px;
+
+  &:hover {
+    background-color: ${(props) => props.hoverBackground};
+    color: #fff;
+  }
+`;
+
+export const CopyablePanelField = memo(
```

Comment:
Can you please move this to its own module.

Reply:
will do